internal/api/router.go

```diff
@@ -38,8 +34,8 @@ func (a *ApiManagerCtx) Mount(r *chi.Mux) {
 	r.Group(a.Http)
 }
 
-func transcodeStart(folder string, profile string, input string) (*exec.Cmd, error) {
-	url, ok := conf.Streams[input]
+func (a *ApiManagerCtx) transcodeStart(folder string, profile string, input string) (*exec.Cmd, error) {
+	url, ok := a.Conf.Streams[input]
 	if !ok {
 		return nil, fmt.Errorf("stream not found")
 	}
```

Comment:
Here we are checking if stream exists twice. Maybe `transcodeStart` should accept URL address instead of stream name. Since we can get it earlier.